gradle-failure-reports/src/test/resources/expected-throwExceptionWithLogs-error-report.xml

```diff
@@ -0,0 +1,32 @@
+<testsuites>
+  <testsuite name=":myProject:throwInnerExceptionWithLogs" tests="1">
+    <testcase name="[:myProject:throwInnerExceptionWithLogs] error: myCustomMessage" className=":myProject:throwInnerExceptionWithLogs">
+      <failure type="ERROR">myCustomMessage
+I have a log line
+
+* Causal chain is:
+	org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':myProject:throwInnerExceptionWithLogs'.
+	org.gradle.api.GradleException: OuterGradleException
+	com.palantir.gradle.failurereports.exceptions.ExceptionWithLogs: myCustomMessage
+	java.lang.RuntimeException: someRuntimeException
+
+* Full exception is:
+org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':myProject:throwInnerExceptionWithLogs'.
+Caused by: org.gradle.api.GradleException: OuterGradleException
+Caused by: com.palantir.gradle.failurereports.exceptions.ExceptionWithLogs: myCustomMessage
+	... PLACEHOLDER_NUMBER more
+Caused by: java.lang.RuntimeException: someRuntimeException
+	... PLACEHOLDER_NUMBER more
+
+</failure>
```

Comment:
doesn't matter too me that much, but curious about the formatting for `</failure>` did you hand generate this or is this expected, I would have thought indentation would be a bit further to the right

Reply:
I think this is because of the multi-line text inside the failure value. Probably a customPrinter might help here, but I don't know if it is worth adding one for now. I am using the [defaultPrettyPrinter](https://github.com/palantir/gradle-failure-reports/blob/develop/gradle-failure-reports/src/main/java/com/palantir/gradle/failurereports/util/XmlResources.java#L34C20-L34C50) atm.